app/src/main/java/org/simple/clinic/addidtopatient/searchresults/AddIdToPatientSearchResultsScreen.kt

```diff
@@ -0,0 +1,134 @@
+import org.simple.clinic.searchresultsview.SearchResultPatientName
+import org.simple.clinic.summary.OpenIntention
+import org.simple.clinic.summary.PatientSummaryScreenKey
+import org.simple.clinic.util.Truss
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.util.identifierdisplay.IdentifierDisplayAdapter
+import org.simple.clinic.util.unsafeLazy
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.UiEvent
+import org.simple.clinic.widgets.hideKeyboard
+import org.threeten.bp.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+class AddIdToPatientSearchResultsScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
+
+  @Inject
+  lateinit var screenRouter: ScreenRouter
+
+  @Inject
+  lateinit var controller: AddIdToPatientSearchResultsController
+
+  @Inject
+  lateinit var utcClock: UtcClock
+
+  @Inject
+  lateinit var identifierDisplayAdapter: IdentifierDisplayAdapter
+
+  private val toolbar by bindView<Toolbar>(R.id.addidtopatientsearchresults_toolbar)
+  private val searchResultsView by bindView<PatientSearchView>(R.id.addidtopatientsearchresults_searchresultsview)
+  private val screenKey by unsafeLazy { screenRouter.key<AddIdToPatientSearchResultsScreenKey>(this) }
+
+  @SuppressLint("CheckResult")
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+    if (isInEditMode) {
+      return
+    }
+    TheActivity.component.inject(this)
+    setupScreen()
+
+    val screenDestroys = RxView
+        .detaches(this)
+        .map { ScreenDestroyed() }
+
+    Observable
+        .mergeArray(
+            screenCreates(),
+            screenDestroys,
+            searchResultClicks(),
+            registerNewPatientClicks()
+        )
+        .observeOn(io())
+        .compose(controller)
+        .observeOn(mainThread())
+        .takeUntil(screenDestroys)
+        .subscribe { it(this) }
+  }
+
+  private fun searchResultClicks(): Observable<UiEvent> {
+    return searchResultsView
+        .upstreamUiEvents
+        .ofType<SearchResultClicked>()
+        .map { AddIdToPatientSearchResultClicked(it.searchResult) }
+  }
+
+  private fun registerNewPatientClicks(): Observable<UiEvent> {
+    return searchResultsView
+        .upstreamUiEvents
+        .ofType<RegisterNewPatient>()
+        .map { AddIdToPatientSearchResultRegisterNewPatient(it.patientName) }
+  }
+
+  private fun setupScreen() {
+    hideKeyboard()
+    toolbar.setNavigationOnClickListener {
+      screenRouter.pop()
+    }
+    toolbar.setOnClickListener {
+      screenRouter.pop()
+    }
+
+    val identifierType = identifierDisplayAdapter.typeAsText(screenKey.identifier)
+    val identifierValue = identifierDisplayAdapter.valueAsText(screenKey.identifier)
+
+    val identifierTextAppearanceSpan = TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body0Left_NumericBold_White100)
+    toolbar.title = Truss()
```

Comment:
The default title provided by the `Toolbar` is single-line, so setting the title on this ends up clipping the text. Can you look at how `AddIdToPatientSearchScreen` is doing this and do the same here?


![Screenshot_1555502577](https://user-images.githubusercontent.com/1978065/56286337-154d6500-6137-11e9-9f0f-a3832ed8c4b7.png)